### PR TITLE
Activated using enter key for sign in - Closes #3740

### LINKED ISSUE #3740: Enter key should be activated in sign in page
### Expected behavior
- Enter key should be activated in sign in page after typing passphrase

### Actual behavior
- After typing in a passphrase you can’t sign in by pressing enter, as the drop down will activate

### Which version(s) does this affect? (Environment, OS, etc...)
v2.0.0

### PR DESCRIPTION
**Pull request by ikem-legend**
### What was the problem?
This PR resolves #3740

### How was it solved?
It was solved by removing the onClick function on the sign in button and adding an onSubmit function to the form

### How was it tested?
Manually

### CODE REVIEW

File: src/components/screens/login/login.js [hunk 1]
```diff
@@ -126,7 +126,7 @@ class Login extends React.Component {
               </p>
             </div>
 
-            <form onSubmit={e => e.preventDefault()}>
+            <form onSubmit={this.onFormSubmit}>
```
Comment (ManuGowda) [inline src/components/screens/login/login.js:129]: Moving it here causes side effects of log in on drop down selection

Comment (ikem-legend) [reply src/components/screens/login/login.js:129]: Fixed

File: src/components/screens/login/login.test.js [hunk 2]
```diff
@@ -1,121 +1,121 @@
...
+  });
+
+  describe('After submission', () => {
+    it('it should call props.login if not already logged with given passphrase', () => {
+      const clipboardData = {
+        getData: () => accounts.delegate.passphrase,
+      };
+      wrapper.find('passphraseInput input').first().simulate('paste', { clipboardData });
+      wrapper.update();
+      wrapper.find('button.login-button').simulate('submit');
```
Comment (ManuGowda) [inline src/components/screens/login/login.test.js:117]: @ikem-legend why this file has fully changed when only this line is changed 🤔 
can you revert the changes and only push the changes to this line?

File: src/components/screens/login/networkSelector/networkSelector.js [hunk 3]
```diff
@@ -1,204 +1,205 @@
...
-          })
-        }
-    </DropdownButton>
-  );
-};
-
-NetworkSelector.displayName = 'NetworkSelector';
-export default NetworkSelector;
+import React, { useRef, useState } from 'react';
+
```
Comment (ManuGowda) [inline src/components/screens/login/networkSelector/networkSelector.js:2]: same with this file as well, somehow the changes are applied to whole file, please make sure to preserve the commit history 🙏🏼
